cql3/statements/describe_statement.cc

```diff
@@ -102,13 +102,16 @@ struct description {
 future<std::vector<description>> generate_descriptions(
     replica::database& db, 
     std::vector<shared_ptr<const keyspace_element>> elements,
-    std::optional<bool> with_internals = std::nullopt) 
+    std::optional<bool> with_internals = std::nullopt,
+    bool skip_sorting = false) 
```

Comment:
Nitpick: I'd avoid the double-negative and call this `bool sort_by_name = true` and then `if (sort_by_name)`.